src/modules/property-panel/styles/sections.scss

```
@@ -55,6 +55,20 @@
   color: #5c5c5c;
 }
 
+.props-textarea {
+  width: 100%;
+  min-height: 2em;
+  max-height: 200px;
+  padding-bottom: 5px;
+  padding-right: 17px;
+  margin-top: 5px;
+  background: transparent;
+  border: 1px solid #e9e9e9;
```

For the sake of consistensy, I would define the border like so: 

```suggestion
  border: none;
  border-bottom: 1px solid #e9e9e9;
```